platform/integrations/ex-integration-guide/exclient-upload-recordings.js

```diff
@@ -47,6 +49,27 @@ async function uploadRecordings(fileName) {
         let response = await client.loginClientCredentialsGrant(clientId, clientSecret);
         logger.verbose('Login successfully');
 // >> START ex-get-presigned-url
+
+        var output = fs.createWriteStream(fileName);
```

Comment:
Shouldn't there be a new quick hit annotation here (e.g. ">> START ex-zip-file"), and then the zip content is in a separate annotation?  And the existing "START ex-get-presigned-url" gets moved down to just above line 73?

...and then on the https://developer.genesys.cloud/platform/integrations/ex-integration-guide/recording-injection-example page we have a new section that references this code segment and talks about zipping the encrypted recording file without using any compression, before the existing upload section.